extensions/azurecore/src/azureResource/commands.ts

```diff
@@ -166,8 +166,24 @@ export function registerAzureResourceCommands(appContext: AppContext, tree: Azur
 		}
 	});
 
-	vscode.commands.registerCommand('azure.resource.openInAzurePortal', async (connectionProfile: azdata.IConnectionProfile) => {
+	vscode.commands.registerCommand('azure.resource.connectazureDataExplorer', async (node?: TreeNode) => {
```

Comment:
This appears to be exactly the same as the `connectsqlserver` command - why not just reuse that?